core/domain/user_domain.py

```diff
@@ -18,12 +18,45 @@
 
 from __future__ import annotations
 
+import datetime
 import re
 
 from core import feconf
 from core import utils
 from core.constants import constants
 
+from typing import List, Optional
+from typing_extensions import TypedDict
+
+
+class UserSettingsDict(TypedDict):
+    """Dictionary representing the UserSettings object."""
+
+    email: str
+    roles: List[str]
+    banned: bool
+    username: Optional[str]
+    normalized_username: Optional[str]
+    last_agreed_to_terms: Optional[datetime.datetime]
+    last_started_state_editor_tutorial: Optional[datetime.datetime]
+    last_started_state_translation_tutorial: Optional[datetime.datetime]
+    last_logged_in: Optional[datetime.datetime]
+    last_created_an_exploration: Optional[datetime.datetime]
+    last_edited_an_exploration: Optional[datetime.datetime]
+    profile_picture_data_url: Optional[str]
+    default_dashboard: str
+    creator_dashboard_display_pref: str
+    user_bio: str
+    subject_interests: Optional[List[str]]
+    first_contribution_msec: Optional[float]
+    preferred_language_codes: Optional[List[str]]
```

Comment:
Ditto as with `subject_interests`.

Reply:
Yeah!, Fixed.
 earlier I missed them accidentally, I apologize for that.